SU-SWS/ace-sdssgryphon
# SDSS-467: footer a and b option issue

# READY FOR REVIEW

# Summary
- Changing the display of the footer lockup.
- It was displaying text even when not requested
- If select None then there is NO LOGO or text
- Widened the area for text
- Fixed up the borders when in sm/xs responsive views.

# Review By (Date)
- soonish

# Urgency
- High

# Steps to Test

1.  checkout the code and refresh.
2. Use a green header site to verify it works
3. In the local footer -> lockup configure page, try the following options: None, Logo, Option A and Option B. 
4. If no text is put in for Option A or B, no text should appear. There should be no white line if there is no text. 
5. Verify that if select none for options, the logo and text disappear.

# Affected Projects or Products
- SDSS subtheme

# Associated Issues and/or People
[- SDSS-467](https://stanfordits.atlassian.net/browse/SDSS-467)
[- SDSS-443](https://stanfordits.atlassian.net/browse/SDSS-443)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
